lib/limits/context.js

```diff
@@ -41,7 +42,15 @@ LimitsContext.prototype.getLimit = function (nodeType, limitName, defaultValue,
         }
         limit = limits[limitName];
     }
-    return { value: limit || defaultValue, message: errorMessage };
+    if (!limit) {
+        // apply default value; a sync. function (with a node parameter) can be used
+        if (typeof defaultValue === 'function') {
```

Comment:
This is a remnant of a previsous iteration. I think I'll remove it, since it's not needed now